inputs/select.vue

```diff
@@ -87,47 +94,58 @@
     props: ['name', 'data', 'schema', 'args'],
     data() {
       return {
+        listOptions: [],
         isDisabled: false
       };
     },
+    mounted() {
+      if(this.args.list){
+        this.fetchListItems().then( listItems => {
+          this.listOptions = listItems;
+        }); // todo: ".catch"?
+      }
+    },
     computed: {
-      safeData() {
-        if (_.isString(this.data)) {
-          return _.find(this.options, (option) => option.value === this.data);
-        } else if (_.isObject(this.data) && !_.isEmpty(this.data)) {
-          return _.reduce(this.options, (safeArray, option) => {
-            if (this.data[option.value] === true) {
-              return safeArray.concat([option]);
-            } else {
-              return safeArray;
-            }
-          }, []);
-        } else {
-          return this.args.multiple ? [] : { value: null, label: 'None' };
+      keys () {
+        return {
+          label: 'name',
+          value: 'value',
+          ...this.args.keys
```

Comment:
this is the problem that circle is hitting. I don't think the object spread operator is supported in the current babel we're using (it's super new!), so you should use something like `_.assign()` instead